src/MCPClient/lib/clientScripts/archivematicaCreateMETS.py

```diff
@@ -33,6 +33,8 @@
 
 sys.path.append("/usr/lib/archivematica/archivematicaCommon")
 from archivematicaFunctions import escape
+from databaseFunctions import getAccessionNumberFromTransfer, getUTCDate
+from elasticSearchFunctions import getDashboardUUID
```

Comment:
It really is.  storageService.py implements a more generic get_setting that is more general than the file it's in.  Move get_setting to archivematicaFunctions and replace getDashboardUUID with a call to that?  Maybe in another PR?